install.php

```diff
@@ -113,9 +113,13 @@
  *
  */
 
-if (version_compare(phpversion(), '7.3.0', '<') === true) {
-    die('ERROR: Whoops, it looks like you have an invalid PHP version. OpenMage supports PHP 7.3.0 or newer.');
```

Comment:
`install.php` is meant to be invoked from the cli, I think that's why there wasn't HTML before.